components/tools/OmeroWeb/omeroweb/webclient/static/webclient/css/layout.css

```diff
@@ -2387,6 +2387,13 @@ div.paging input.button_pagination {
     right: 0px;
     overflow: auto;
 }
+.zoomLabel {
+	font-size: 12px;
```

Comment:
weird indent

Reply:
@will-moore could fix that so we can merge